promtimer/annotations.py

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2020 Couchbase, Inc All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from os import path
+import time
+import json
+from dateutil import parser as dateparser
+import urllib.request
+
+ANNOTS_URL = 'http://localhost:13300/api/annotations'
```

Comment:
Don't hard code 13300. This can be set via the --grafana-port command -- you'll need to pick it up from there.